### ISSUE TITLE
Generate fixture Item covers

### ISSUE THREAD

**Issue by kittsville** [kittsville/WP-Librarian-Fixtures#9]
Currently all fixture items generated have no cover image. This could be remedied using images procedurally generated with variable size and content. There may be lightweight existing packages out there or a solution to pump noise into an image file may work just as well.

This is something to be explored when existing issues feel a bit dull and an interesting new challenge feels appropriate. Otherwise low priority should be given to this issue.

**Comment by kittsville** [comment]
This is a bit of a problem. [GeoPattern](https://github.com/redeyeventures/geopattern-php) is a wonderful library that procedurally generates tiling patterns from string inputs. GitHub uses it to generate the patterns in the headers of their guides, using the guide titles as the input string (neat, huh?). Here's this issue's title as a pattern:
![desktopy](https://cloud.githubusercontent.com/assets/6013087/8501511/1f137ec6-219e-11e5-927a-bcc14842fd63.png)

However, this library generates SVG patterns, which are not natively supported by WordPress. I could use some hooks to add SVG support but it would seem silly to be testing WP-Librarian on SVG item covers when it'll be dealing with PNG, JPG and GIF covers (only potentially handling SVG covers if a dev added custom support to their library).

The alternative is converting the SVG images to PNGs. This is possibly the worst option as it would either require devs installing Imagick or using the user's browser and HTML5 canvas as an intermediate converter. This first option requires the user jumping through a bunch of hoops, defeating the point of the Dev Kit which is supposed to be an easy way to start WP-Librarian development on a new computer. The latter would be a security risk and very browser reliant. While those could be waved away in that it's a dev kit running on a web developer's computer still doesn't seem like a good way of going about things.

tl;dr This is why we can't have nice things :sob:
